### PR TITLE
Add Cog file to build Docker image

### PR DESCRIPTION
**Pull request by andreasjansson**
Hi @orpatashnik!

This pull request makes it possible to run StyleCLIP inside a Docker environment, which makes it easier for other people to run it.  We're using an open source tool called [Cog](https://github.com/replicate/cog) to make this process easier.

This also means we can make a web page where other people can try out your model! View it here: https://replicate.ai/orpatashnik/styleclip

That page also has instructions on how to use the Docker image, which is on our registry at r8.im/orpatashnik/styleclip

### CODE REVIEW

File: global_directions/manipulate.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import numpy as np
 import tensorflow as tf
 from dnnlib import tflib
-from global_directions.utils.visualizer import HtmlPageVisualizer
+from utils.visualizer import HtmlPageVisualizer
```
Comment (orpatashnik) [inline global_directions/manipulate.py:9]: This line fails when running the global directions notebook, why is this change necessary?

File: predict.py [hunk 2]
```diff
@@ -0,0 +1,196 @@
+import copy
```
Comment (orpatashnik) [inline predict.py:1]: Is it possible to locate this file in other directory or replicate requires it to be in the root directory?

**Review by orpatashnik — COMMENTED** [review]
Please see my comments :blush: 
Ideally, I think it is better to put the replicae-related files in a dedicated directory, but it is not necessary.

File: global_directions/manipulate.py (hunk 1 above)
Comment (andreasjansson) [reply global_directions/manipulate.py]: I made it work without this change, so I've reverted back to the original line.

File: predict.py (hunk 2 above)
Comment (andreasjansson) [reply predict.py]: Unfortunately Cog doesn't support predictors in subdirectories yet. We have an [open issue](https://github.com/replicate/cog/issues/164) for it but for now I've renamed `predict.py` -> `cog_predict.py`.

**Pull request merged**